### PR TITLE
Feature/eoi/creation of folders for eoi

### PR DESCRIPTION
**Pull request by ulysses-cognizant**
### JIRA link
Base folder structure for EOI

### Change description
Fixed conflicts 

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Check and send page updated
- [ ] UI changes look good on mobile

### Developer self-QA run statement

- [x ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x ] No

### CODE REVIEW

**Review by CesarPerez — APPROVED** [review]

**Review by prathap-cognizant — APPROVED** [review]

**Review by mohanraja-cognizant — APPROVED** [review]

**Review by AyeshaFarhin — APPROVED** [review]
Approved

File: src/main/public/assets/scripts/validations/rfi_acronyms.js [hunk 1]
```diff
@@ -198,8 +167,8 @@ const emptyFieldCheck = () => {
         errorStore.push(fieldCheck);
       }
       if (term_field.value.trim() === '' && definition_field.value.trim() !== '') {
-        ccsZaddErrorMessage(term_field, 'You must add information in both fields.');
-        fieldCheck = [term_field.id, 'You must add information in both fields.'];
+        ccsZaddErrorMessage(term_field, 'Please provide a term or acronym');
```
Comment (daimdev) [inline src/main/public/assets/scripts/validations/rfi_acronyms.js:170]: it's consist of old version

File: src/main/public/assets/scripts/validations/rfi_acronyms.js [hunk 2]
```diff
@@ -31,42 +31,11 @@ document.addEventListener('DOMContentLoaded', () => {
...
-      const end = 10;
-
-      for(var a = start; a <= end; a++){
-        let input = $(`#rfi_term_${a}`)
-        let textbox = $(`#rfi_term_definition_${a}`);
-       
-
-        if(input.val() !== ""){
-
-          $(`#rfi_term_${a}-error`).remove();
```
Comment (daimdev) [inline src/main/public/assets/scripts/validations/rfi_acronyms.js:48]: it's consist of old version

File: src/main/features/rfi/views/questions.njk [hunk 3]
```diff
@@ -430,7 +430,7 @@
 
                 <div class="govuk-button-group">
                   {{ CCSButton({
-                        text: "Save and continue",
```
Comment (daimdev) [inline src/main/features/rfi/views/questions.njk]: it's consist of old version

File: src/main/features/rfi/controller/type.ts [hunk 4]
```diff
@@ -1,62 +1,58 @@
...
  * @GETController
  */
 
 export const POST_TYPE = (req: express.Request, res: express.Response) => {
-   const { agreement_id } = req.query;
-   const projectId = req.session['projectId']
-   const event_id = req.session['eventId'];
-
-
-
```
Comment (daimdev) [inline src/main/features/rfi/controller/type.ts:41]: it's consist of old version

File: src/main/views/components/textarea/template.njk [hunk 5]
```diff
@@ -37,7 +37,7 @@
     visuallyHiddenText: params.errorMessage.visuallyHiddenText
   }) | indent(2) | trim }}
 {% endif %}
- <textarea class="govuk-textarea {{- ' govuk-textarea--error' if params.errorMessage }} {{- ' ' + params.classes if params.classes}}" id="{{ params.id }}" name="{{ params.name }}" rows="{%if params.rows %} {{- params.rows -}} {% else %}5{%endif %}"
+  <textarea class="govuk-textarea {{- ' govuk-textarea--error' if params.errorMessage }} {{- ' ' + params.classes if params.classes}}" id="{{ params.id }}" name="{{ params.name }}" rows="{%if params.rows %} {{- params.rows -}} {% else %}5{%endif %}"
```
Comment (daimdev) [inline src/main/views/components/textarea/template.njk:40]: it's consist of old version

File: src/main/features/rfi/controller/questions.ts [hunk 6]
```diff
@@ -158,21 +153,17 @@ export const POST_QUESTION = async (req: express.Request, res: express.Response)
...
-          if (object_values.length > 0) {
+          if(object_values.length > 0){
             const object_values_Keyterm = object_values[0].value;
             const object_values_acronyms = object_values[1].value;
-            const keyTermsAcronymsSorted = [];
-            for (let start = 0; start < object_values_Keyterm.length; start++) {
-              const termAndAcryonys = {
-                value: object_values_Keyterm[start],
-                text: object_values_acronyms[start],
-                selected: true,
```
Comment (daimdev) [inline src/main/features/rfi/controller/questions.ts:169]: it's consist of old version

**Review by daimdev — CHANGES_REQUESTED** [review]

**Comment by ulysses-cognizant** [comment]
Will create new branch to resolve issues above.

File: src/main/features/rfi/controller/questions.ts (hunk 6 above)
Comment (ulysses-cognizant) [reply src/main/features/rfi/controller/questions.ts:169]: fixing it.

File: src/main/views/components/textarea/template.njk (hunk 5 above)
Comment (ulysses-cognizant) [reply src/main/views/components/textarea/template.njk:40]: fixing it.

File: src/main/features/rfi/controller/type.ts (hunk 4 above)
Comment (ulysses-cognizant) [reply src/main/features/rfi/controller/type.ts:41]: fixing it.

File: src/main/features/rfi/views/questions.njk (hunk 3 above)
Comment (ulysses-cognizant) [reply src/main/features/rfi/views/questions.njk]: fixing it.

File: src/main/public/assets/scripts/validations/rfi_acronyms.js (hunk 2 above)
Comment (ulysses-cognizant) [reply src/main/public/assets/scripts/validations/rfi_acronyms.js:48]: fixing it.

File: src/main/public/assets/scripts/validations/rfi_acronyms.js (hunk 1 above)
Comment (ulysses-cognizant) [reply src/main/public/assets/scripts/validations/rfi_acronyms.js:170]: fixing it.

**Pull request closed without merging**